docs/docs/integrations/document_loaders/cratedb.ipynb

```diff
@@ -0,0 +1,232 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CrateDB\n",
+    "\n",
+    "This notebook demonstrates how to load documents from a [CrateDB] database,\n",
+    "using the [SQLAlchemy] document loader.\n",
+    "\n",
+    "It loads the result of a database query with one document per row.\n",
+    "\n",
+    "[CrateDB]: https://github.com/crate/crate\n",
+    "[SQLAlchemy]: https://www.sqlalchemy.org/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prerequisites"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#!pip install crash 'langchain[cratedb]'"
```

Comment:
That's no longer a valid installation incantation. Something like this would be better:

```suggestion
    "#!pip install crash langchain sqlalchemy-cratedb"
```